crates/gpui/src/image_cache.rs

```diff
@@ -63,7 +72,16 @@ impl From<Arc<PathBuf>> for UriOrPath {
     }
 }
 
-type FetchImageTask = Shared<Task<Result<Arc<ImageData>, Error>>>;
+#[derive(Clone)]
+pub enum FetchImageType {
+    Raster(Arc<ImageData>),
+    Vector {
```

Comment:
We should remove this type and have the SVG rasterization generate an ImageData.

Reply:
agreed, makes sense.

Reply:
done, let me know what you think